recipes/blast/meta.yaml

```diff
@@ -10,43 +10,57 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - boost
     - zlib
     - bzip2
-    - gnutls
-    - gmp
-    - nettle 3.4
     - pcre
+    - entrez-direct
```

Comment:
What does this one include, actually?

Reply:
This is needed for `get_species_taxids.sh`